draft-ietf-quic-qlog-quic-events.md

```diff
@@ -1824,8 +1814,9 @@ PathResponseFrame = {
 
 ### ConnectionCloseFrame
 
-raw_error_code is the actual, numerical code. This is useful because some error
-types are spread out over a range of codes (e.g., QUIC's crypto_error).
+error_code_value is the actual, numerical code (the parsed VLIE-encoded
+integer). This is useful because some error types are spread out over a range of
+codes (e.g., QUIC's crypto_error).
```

Comment:
I find this a bit ambigous to read. How about something like

```suggestion
The error_code_value is the numerical value without VLIE encoding.
This is useful because some error types are spread out over a range of
codes (e.g., QUIC's crypto_error).
```